shared-libs/validation/test/validations.js

```diff
@@ -248,6 +248,61 @@ describe('validations', () => {
     });
   });
 
+  it.only('should fail uniquePhone if doc is found in db', () => {
```

Comment:
Did you mean to check `phone` uniqueness here? If so can you format the test name like: 
`should fail uniquePhone if phone is found in db`

Reply:
Thanks. Done.